edx_django_utils/hooks/exceptions.py

```diff
@@ -0,0 +1,15 @@
+"""
+Exceptions thrown by Hooks.
+"""
+
+
+class HookException(Exception):
+    """
+    Base exception for hooks. It is re-raised by the hook executor.
+    """
+
+    def __init__(self, message='', redirect_to=None, error_code=None):
+        super().__init__()
+        self.message = message
```

Comment:
should we add a prefix to message? @morenol

Reply:
I think that that message could be rendered in the templates, for instance in case of error in the pre_login filter. It would be weird if the message rendered in the template starts with HookException"